src/content/codelabs/dart-cheatsheet.md

```diff
@@ -1728,7 +1728,7 @@ void main() {
 ## Factory constructors
 
 Dart supports factory constructors,
-which can return subtypes or even null.
+which can return instances of a class or even subtypes.
```

Comment:
Everything is an instance of a class in Dart (even `null`), so that doesn't say much. But you're exactly right that factory constructors can't return `null` anymore. That's a good catch. How about:

"which can return an instance of the surrounding class or even a subtype of it."